README.md

```diff
@@ -17,6 +65,15 @@ cosmoz-moment is a Polymer component for centralized management of Moment.js wit
 <link rel="import" href="bower_components/cosmoz-moment/cosmoz-moment.html" />
 ```
 
+### Use it inside your element
```

Comment:
cosmoz-moment should only be used once per _app_ to manage the locale and moment.js.
cosmoz-moment-timeago can be used within elements, or directly in the app, multiple times.
Another (better?) use case is to implement the Cosmoz.MomentBehavior in an element to be able to compute on `locale` and use moment.js-helpers.